README.md

```diff
@@ -1,6 +1,6 @@
 # Dokku [![Build Status](https://img.shields.io/circleci/project/dokku/dokku/master.svg?style=flat-square "Build Status")](https://circleci.com/gh/dokku/dokku/tree/master) [![Ubuntu Package](https://img.shields.io/badge/package-ubuntu-brightgreen.svg?style=flat-square "Ubuntu Package")](https://packagecloud.io/dokku/dokku) [![IRC Network](https://img.shields.io/badge/irc-freenode-blue.svg?style=flat-square "IRC Freenode")](https://webchat.freenode.net/?channels=dokku) [![Slack Group](https://img.shields.io/badge/irc-slack-blue.svg?style=flat-square "Slack Group")](https://glider-slackin.herokuapp.com/) [![Documentation](https://img.shields.io/badge/docs-viewdocs-blue.svg?style=flat-square "Viewdocs")](http://dokku.viewdocs.io/dokku/) [![Gratipay](https://img.shields.io/gratipay/dokku.svg?style=flat-square)](https://gratipay.com/dokku/)
-+[![OpenCollective](https://opencollective.com/dokku/sponsors/badge.svg)](#sponsors)
-+[![OpenCollective](https://opencollective.com/dokku/backers/badge.svg)](#backers) 
+[![OpenCollective](https://opencollective.com/dokku/sponsors/badge.svg)](#sponsors)
+[![OpenCollective](https://opencollective.com/dokku/backers/badge.svg)](#backers) 
 
```

Comment:
Oh this is fine now, the previous ones were separate from these. These are part of the "diff" (notice the `-` on the red bit? Same thing, but in reverse.).